lib/temping.rb

```diff
@@ -21,7 +21,7 @@ def teardown(clear_dependencies: false)
           end
         end
         @model_klasses.clear
-        ActiveSupport::Dependencies.clear if clear_dependencies
+        ActiveSupport::Dependencies::Reference.clear!
```

Comment:
Hey, I'm a user of Temping, and I very strongly feel that this check should not be kept.

The check was introduced as a workaround to a bug in the dependency clearing. It wasn't motivated by a need to avoid dependency clearing, but to avoid triggering that bug. Assuming all bugs were fixed, why _wouldn't_ you want to clear the dependencies? Not clearing them is, itself, a bug IMO, because it would a method called `teardown` that purposefully left behind persistent state!

As a result of this option being introduced and dependency clearing being disabled by default, I lost _hours_ of my time running a debugger through the Temping source code until I found the line that checked for that option being passed.

So, I think the check should be removed because clearing the dependencies in teardown is objectively the correct behaviour. There should be no reason at all why you'd want to disable it, and having it disabled by default leads to surprising and frustrating behaviour that takes a long time to debug.